lib/packer/manifest.rb

```diff
@@ -45,11 +45,11 @@ def handle_missing_entry(name)
 
     def missing_file_from_manifest_error(bundle_name)
       <<~MSG
-        Webpacker can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
+        Packer can't find #{bundle_name} in #{config.public_manifest_path}. Possible causes:
         1. You want to set packer.yml value of compile to true for your environment
           unless you are using the `webpack -w` or the webpack-dev-server.
         2. webpack has not yet re-run to reflect updates.
-        3. You have misconfigured Packer's webpacker.yml file.
+        3. You have misconfigured Packer's packer.yml file.
```

Comment:
Could we add something about running `yarn install` here? Sounds like a potential pitfall.

Reply:
Wouldn't this be caught by a compilation error is it will not be able to find a dependency?
On this note should we consider running explicitly `yarn install` as part of a deployment?
I will add the comment anyways.

Reply:
Yarn install gets run by Rails itself during deployment but it's a potential issue with Sinatra apps.